pkg/local_object_storage/metabase/VERSION.md

```diff
@@ -57,6 +57,10 @@ Numbers stand for a single byte value.
   - Name: container ID + `9`
   - Key: object ID
   - Value: marshaled object
+- Buckets containing object or LINK type
+  - Name: container ID + `18`
```

Comment:
I thought they're sorted here.

Reply:
they have a logical title (`Unique index buckets`) so i decided them to be together